server/migrations/20200608192629-add-discussion-drafts.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   up: (queryInterface, DataTypes) => {
     return queryInterface.createTable('DiscussionDrafts', {
       id: { type: DataTypes.INTEGER, autoIncrement: true, primaryKey: true },
-      address_id: { type: DataTypes.INTEGER, allowNull: false, references: { model: 'Addresses', key: 'id' } },
+      author_id: { type: DataTypes.INTEGER, allowNull: false, references: { model: 'Addresses', key: 'id' } },
```

Comment:
I changed author_id to address_id for consistency (and since the comments on the PR were unaddressed at the end of last week). If you haven't been able to replicate this error, let's just leave this fix out of this PR now.

Reply:
Ah I see. I was actually hitting the same migration error as Zak, which is why I switched. Should I remove the function in the migration that changes address_id to author_id on both drafts and threads? Or just leave it?

Reply:
Not really sure what you mean; there was a migration to add discussion drafts, that included an author_id, and a migration a bit later to overwrite that (as well as offchain threads) to having an address_id. I don't have access to latest.dumps other than what gets passed down to me every once'n'awhile, so I'm probably working off an old version, but that db is up to speed on neither migration.

Reply:
I'm just wondering what caused DiscussionDrafts and OffchainThreads to have address_id instead of author_id. But it seems they had address_id all along, so everything is fine.